templating/twig_extension_repository.rst

```diff
@@ -0,0 +1,93 @@
+.. index::
+   single: Using the Twig Extensions Repository
```

Comment:
You're right in that there is nothing special with these extensions.

However, all Twig Extensions I normally use are either part of a bundle or there is a bundle to set them up in Symfony.

I felt there was kind of a gap for the "official" Twig Extensions. They're mentioned in the Symfony docs, but every time I need them it takes me moment to remind myself that I have to set them up that way - which is why I wrote this article to help the future me :-).